toc/ROLES.md

```diff
@@ -49,6 +49,20 @@ establish sub-working groups. Working groups delegate change approval to Approve
     </tr>
   </thead>
 
+  <tr>
+    <td>(none)</td>
+    <td>None</td>
+    <td>
+        <p>Signed CLA</p>
```

Comment:
Technically speaking, a member does only need a signed CLA if they submit PRs.

Having said this, "Signed CLA" is a quality that does fit the "Responsibility" column quite well.

Reply:
Feedback from my team: CLA signing is actually seen as a 'requirement'.
Maybe like this: "Signed CLA (for submitting PRs)

Reply:
My bad, it _is_ proposed for the requirements column. Changed my comment accordingly. I'd still say it applies only for one of the priviledges.

Reply:
I think I fixed this!